src/components/latest-scanned.tsx

```
@@ -13,44 +14,83 @@ import {
+            <Skeleton className='w-[250px] h-4' />
+            <Skeleton className='w-[200px] h-4' />
+          </div>
+        )}
+
+        {!isLoading && !recentScans?.length && (
+          <p className='text-center text-muted-foreground'>
+            Belum ada data kehadiran
+          </p>
+        )}
+
+        {!isLoading && recentScans!?.length > 0 && (
```

Biome

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)